transport/http/handler.go

```diff
@@ -98,7 +99,17 @@ func (h handler) callHandler(w http.ResponseWriter, req *http.Request) error {
 
 	start := time.Now()
 
-	// TODO capture and handle panic
+	// We recover panics from the user handler.
+	defer func() {
+		if r := recover(); r != nil {
+			if err, ok := r.(error); ok {
+				returnErr = err
+			} else {
+				returnErr = fmt.Errorf("%q", r)
```

Comment:
I think we should wrap errors/messages from panic, so we have a message that looks like:
`panic handling SomeMethod: %v`